compiler/src/dotty/tools/dotc/util/NameTransformer.scala

```diff
@@ -41,6 +41,18 @@ object NameTransformer {
   enterOp('?', "$qmark")
   enterOp('@', "$at")
 
+
+  /* Encode characters that are illegal */
+  def encodeSting(value: String): String = {
```

Comment:
There's a typo (the `r` is missing from `String`)

Reply:
good catch :+1: